src/lib.rs

```diff
@@ -63,6 +63,16 @@ cfg_if::cfg_if! {
   }
 }
 
+mod hawktracer {
+  cfg_if::cfg_if! {
+    if #[cfg(feature="tracing")] {
+      pub use rust_hawktracer::*;
+    } else {
+      pub use noop_proc_macro::hawktracer;
```

Comment:
The macros don't touch the method if profiling is off so this might not be needed.
See: https://github.com/AlexEne/rust_hawktracer_proc_macro/blob/589a7070f9415497b946f6dee3d8d19c9b1fe0a3/src/lib.rs#L33

Reply:
Thank you :)